code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1789,3 +1800,19 @@
 
 /mob/living/carbon/human/get_mob_riding_slots()
 	return list(back, head, wear_suit)
+
+/mob/living/carbon/human/proc/update_fullness()
+	var/list/new_fullness = list()
+	vore_fullness = 0
+	for(var/belly_class in vore_icon_bellies)
+		new_fullness[belly_class] = 0
+	for(var/obj/belly/B as anything in vore_organs)
+		new_fullness[B.belly_sprite_to_affect] += B.GetFullnessFromBelly()
+	for(var/belly_class in vore_icon_bellies)
+		new_fullness[belly_class] /= size_multiplier //Divided by pred's size so a macro mob won't get macro belly from a regular prey.
+		new_fullness[belly_class] = round(new_fullness[belly_class], 1) // Because intervals of 0.25 are going to make sprite artists cry.
+		vore_fullness_ex[belly_class] = min(vore_capacity_ex[belly_class], new_fullness[belly_class])
+		vore_fullness += new_fullness[belly_class]
+	vore_fullness = min(vore_capacity, vore_fullness)
+	update_vore_belly_sprite()
+	update_vore_tail_sprite()
```

Comment:
moved from mob to human, compare with our proc override on human... likely comment out as well